fastlane-plugin-wpmreleasetoolkit.gemspec

```diff
@@ -61,4 +60,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'rubocop-require_tools', '~> 0.1.2'
   spec.add_development_dependency 'rubocop-rspec', '2.3.0'
   spec.add_development_dependency 'simplecov', '~> 0.16.1'
+  spec.metadata['rubygems_mfa_required'] = 'true'
```

Comment:
This was automatically added, but sounds like a bigger change in our process that is outside the scope of this PR.

I did a bit of reading in the RubyGem docs here about this specific line: https://guides.rubygems.org/mfa-requirement-opt-in/

While it would increase security, I'm guessing that we'd need to have a discussion about how and if to implement MFA for our gem publishing.

Reply:
Yeah, especially since our gem publishing is automated by CI every time we create a new tag on this repo, and I doubt this process would work well with a OTP being asked during that process 😓

Reply:
I added a Rubocop exception for this until we decide whether to pursue it further. https://github.com/wordpress-mobile/release-toolkit/pull/464/commits/cdfa2baf90f94d34ec32b83d029020bc4715f9c0